cookbooks/crystal_preferred_orientation_olivine_fraters_billen_2021/doc/crystal_preferred_orientation_olivine_fraters_billen_2021.md

```diff
@@ -0,0 +1,132 @@
+# Olivine Fabric Developments Under Simple Shear
+
+*This section was contributed by Xiaochuan Tian, with helps from Yijun Wang and Menno Fraters. It is
+based on a section in {cite:t}`fraters_billen_2021_cpo`
+by Menno Fraters and Magali Billen published in 2021.*
+
+This cookbook explains how to set up a numerical experiment for fabric
+developments of a single olivine particle under simple shear macroscopic strain.
+It uses ASPECT's
+crystal preferred orientation (CPO) implementation, which is described in detail in
+{cite:t}`fraters_billen_2021_cpo` and the paper is open accessed at
+[here](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2021GC009846).
```

Comment:
The paper is open access in general, so people can just follow the doi in the reference. I don't think there needs to be a separate link here.